server/models/likes.js

```diff
@@ -1,6 +1,12 @@
 'use strict';
 module.exports = function(sequelize, DataTypes) {
   var likes = sequelize.define('Likes', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
```

Comment:
/Users/chloeechikson/GH/deco/server/models/likes.js:8
      type: Sequelize.INTEGER
            ^

ReferenceError: Sequelize is not defined